src/main.sh

```diff
@@ -106,9 +110,9 @@ configureCredentials () {
         ${vault_address}/v1/auth/approle/login | jq -r .auth.client_token)
         echo "export VAULT_TOKEN=${VAULT_TOKEN}" >> env_vars
       /usr/local/bin/vault read -format=json secret/dsde/datarepo/dev/sa-key.json | \
-        jq .data > jade-dev-account.json
-      jq -r .private_key jade-dev-account.json > jade-dev-account.pem
-      chmod 600 jade-dev-account.pem
+        jq .data > ${google_application_credentials}
+      jq -r .private_key ${google_application_credentials} > ${google_application_credentials_pem}
+      chmod 600 ${google_application_credentials_pem}
```

Comment:
I think these should all use the uppercase version of this variable, and the lowercase one should be removed.

Reply:
Will address in follow up ticket.